src/main/java/org/broadinstitute/hellbender/utils/Utils.java

```diff
@@ -593,13 +593,13 @@ public static String calculateFileMD5( final File file ) throws IOException{
 
     /**
      * Checks that the collection does not contain a {@code null} value (throws an {@link IllegalArgumentException} if it does).
-     * The implementation calls {@code c.contains(null)} to determine the presence of null.
      * @param c collection
      * @param message the text message that would be pass to the exception thrown when c contains a null.
      * @throws IllegalArgumentException if a {@code o == null}
      */
     public static void containsNoNull(final Collection<?> c, final String message) {
-        if (c.contains(null)){
+        Utils.nonNull(c, message);
```

Comment:
(github will not let me comment on line 598 `* @throws IllegalArgumentException if a {@code o == null}` but that line makes no sense